src/tests/ftest/pool/evict_test.py

```diff
@@ -86,13 +87,16 @@ def evict_badparam(self, test_param):
             "Pool UUID: %s\n Pool handle: %s\n Server group: %s\n",
             self.pool.uuid, self.pool.pool.handle.value, self.pool.name)
 
-        if test_param == "BAD_SERVER_NAME":
-            # Attempt to evict pool with invalid server group name
-            # set the server group name directly
-            self.pool.pool.group = ctypes.create_string_buffer(test_param)
-            self.log.info(
-                "Evicting pool with invalid Server Group Name: %s", test_param)
-        elif test_param == "invalid_uuid":
+        # Commented out due to DAOS-5545
```

Comment:
Could you create a story to re-enable it and then link it to DAOS-5545?

Reply:
@berserk-fury Sure, I was planning to do it once this PR is approved. Do you want me to create it before this?

Reply:
@berserk-fury I created the ticket DAOS-6356 requesting this.
https://jira.hpdd.intel.com/browse/DAOS-6356
Thank you!